src/main/resources/logback.xml

```diff
@@ -10,12 +10,6 @@
         </then>
```

Comment:
Sentry log appender is automatically configured when used with sentry spring boot

Reply:
Oh interesting.  Great job getting to the bottom of it.  I'm surprised it works for the other services

Reply:
I agree, I don't understand either. This has been part of the puzzle, for sure.